cedar-policy/src/frontend/is_authorized.rs

```diff
@@ -1398,4 +1559,154 @@ mod test {
         }"#;
         assert_is_failure(&json_is_authorized(call), true, "found duplicate key");
     }
+
+    #[cfg(feature = "partial-eval")]
```

Comment:
It looks like in Rust individual files and directories are implicit modules however it doesn't seem like the `#[cfg(feature = "...")]` construct can be used at the file level. These's an explicit `mod` keyword that can presumably be used to define sub-modules with a single file but then we would end up with `cedar_policy::frontend::is_authorized::partial::json_is_authorized_partial` as the public method signature.

Reply:
I think Andrew is suggesting that the tests could be a module like

```
#[cfg(feature = "partial-eval")]
#[cfg(test)]
mod test_partial {
...
}
```

Reply:
Ah, that makes sense. I thought that applied to all diffs behind the feature flag.

Reply:
> it doesn't seem like the `#[cfg(feature = "...")]` construct can be used at the file level

It can, you need to add `!`, like so: `#![cfg(feature = "...")]`